tools/downloader/src/open_model_zoo/model_tools/converter.py

```diff
@@ -120,13 +144,26 @@ def main():
         requested_precisions = set(args.precisions.split(','))
         unknown_precisions = requested_precisions - _common.KNOWN_PRECISIONS
         if unknown_precisions:
+            try:
+                telemetry.send_event('md', 'converter_error', str({'error': 'Unknown precisions specified.', 'model': 'none'}))
+                telemetry.end_session('md')
+                telemetry.force_shutdown(1.0)
+            except NameError:
+                pass
             sys.exit('Unknown precisions specified: {}.'.format(', '.join(sorted(unknown_precisions))))
 
     models = _common.load_models_from_args(parser, args)
 
     output_dir = args.download_dir if args.output_dir is None else args.output_dir
 
     def convert(reporter, model):
+        try:
+            telemetry = tm.Telemetry()
```

Comment:
Not sure what the point of this is - `telemetry` should still be accessible from the outer function scope.

Reply:
Removed